### ISSUE TITLE
Flutterflow Editor Switches back to Light Mode during development

### ISSUE THREAD

**Issue by Dev-Muhammad-Junaid** [FlutterFlow/flutterflow-issues#3628]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I am building the app and whenever I switch to Dark Mode the app reverts itself back again to light mode after few seconds.
Not sure why. This is happening in the Flutterflow dashboard not the actual test/run mode setup.

### Expected Behavior

I expect the app to stay in the light mode or dark mode if I am switching

### Steps to Reproduce

1. Simply switch the mode from dark to light from flutterflow editor.
2. Do Nothing or move across different screens
3. The app will auto switch its mode after some time

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFfl8n6z5N2m8tJ1a6ILcFvlikWJn0cTIMziNZ+exUbfaL1Bu0ies7SQxZCZNOoaVNfMWKYpGUF+s7rjuzpCPEqOTKeQIM4/pFQeA3Oe2iuR7rRPZCoVURCI51PIWqG4aWNhAl/JuhicloC6l+MN+fLSjHDFOu/Zwh9f6PHaOI=

### Visual documentation

.

### Environment

```markdown
- FlutterFlow version: v4.1.81+ released August 7, 2024
- Platform: MacOS, Web
- Browser name and version: Chrome
- Operating system and version affected:  MacOS
```

### Additional Information

.

**Comment by rzambroni** [comment]
Thanks for the report. That’s a very strange issue. I wasn’t able to replicate it on either the macOS version or the web app.

Does this happen consistently on your end? if it is possible and it is just after a few seconds, would you mind providing a video recording of it happening?

Thanks so much!

**Comment by MartinLockheart** [comment]
Adding to this: My dev partner and I noticed that if we switched light mode/dark mode in Flutterflow while on the same project, it flipped the switch for the other one too. This happened a while back, so I don't know if it's a current issue, but if @Dev-Muhammad-Junaid has someone else working with him on the same project, that might narrow things down.

**Comment by rzambroni** [comment]
that could definitely be the case and would answer why we couldn't replicate the problem. Will take another look with the collaboration of a team member.

**Comment by Dev-Muhammad-Junaid** [comment]
Yes, I am added a collaborator, so this could help with identifying the issue.Thanks for the update @MartinLockheart and @rzambroni

**Comment by rzambroni** [comment]
Upon investigating with a collaborator we were able to replicate the issue. I'll send this to the ENG team so they can take a look. 

Thank you both for the assistance.

**Comment by Dev-Muhammad-Junaid** [comment]
Glad to know its identified
